docker-tool.sh

```diff
@@ -17,15 +17,24 @@
 #   specific language governing permissions and limitations
 #   under the License.
 
+stop_and_die() {
+  docker buildx stop tika-builder || die "couldn't stop builder -- make sure to stop the builder manually! "
```

Comment:
I'm make this `docker buildx rm` instead.  Just stopping it seems to have the same effect, complaints about the existing instance